docs/src/assets/readme.md

```diff
@@ -0,0 +1,439 @@
+# Source
```

Comment:
As commented - I would avoid adding this much detail.

Just write something like

```
# Introduction

In this folder we store the following data sets:

* german_credit.csv
* iris.csv

# German Credit data set

## License:
link

## Source:

both info that the original data is from UCI, and that this data is from Kaggle (as this information is missing)
```

If you feel uncomfortable with something just leave an empty space and I will fill it

Reply:
Oh okay, i had not read your above comment.